web/app.js

```diff
@@ -41,6 +41,12 @@ import { SecondaryToolbar } from 'pdfjs-web/secondary_toolbar';
 import { Toolbar } from 'pdfjs-web/toolbar';
 import { ViewHistory } from 'pdfjs-web/view_history';
 
+var {
```

Comment:
I looked through `debugger.js` quickly, and it seems that it only uses *two* of the `pdfjsLib` properties. Hence I've changed the initialization to only pass in those when calling `PDFBug.init` instead, and subsequently this point is now fixed :-)